@xen-orchestra/backups/RemoteAdapter.js

```diff
@@ -448,9 +453,22 @@ class RemoteAdapter {
     return backupsByPool
   }
 
-  async listVmBackups(vmUuid, predicate) {
+  async invalidateVmBackupListCache(vmUuid) {
+    try {
+      await this.handler.unlink(`${BACKUP_DIR}/${vmUuid}/cache.json.gz`)
+
+      // remove any pending loc
+    } catch (error) {
+      if (error.code === 'ENOENT') {
```

Comment:
Do you think `Handler#unlink()` should handle it like it does for `Handler#rmdir()`?

Reply:
I am not sure : in most case we are deleting a file that should be there ( like a backup after a listing), and I do prefer a explicit error message givin to the user 
it looks like `invalidateVmBackupListCache` is the only method that use unlink and don't care if it's not here

Reply:
Oh ok, you found it :joy:

Reply:
Already handled by `Handler#unlink()`.